### PR TITLE
lesson 2 assignments

### PR DESCRIPTION
**Pull request by yixingxu**

### CODE REVIEW

**Comment by yixingxu** [comment]
finished the second assignment

File: students/yixingxu/lesson02/Fizz-Buzz.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for i in range(1,100):
+	if i%3 ==0 and i%5 ==0:
+		print('FizzBuzz')
+	elif i%3 ==0:
+		print('Fizz')
+	elif i%5 ==0:
+		print('Buzz')
+	else: print(i)
+
```
Comment (natasha-aleksandrova) [inline students/yixingxu/lesson02/Fizz-Buzz.py]: your code is over-indented; per pep8 guidelines indent should be exactly 4 spaces (not tabs).

you should be able to configure whichever editor you are using to indent correctly for you.

File: students/yixingxu/lesson02/Fizz-Buzz.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+for i in range(1,100):
```
Comment (natasha-aleksandrova) [inline students/yixingxu/lesson02/Fizz-Buzz.py]: python modules (file names) follow convention of snake_case naming -> `fizz_buzz.py`

File: students/yixingxu/lesson02/series.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	for i in range(10):		
+		print(sum_series(i))
+		
+	print('lucas number:')	
+	for i in range(10):		
+		print(sum_series(i,2,1))
+		
+	print('series number starting with 4 and 6:')	
+	for i in range(10):
+		print(sum_series(i,4,6))
```
Comment (natasha-aleksandrova) [inline students/yixingxu/lesson02/series.py]: it is a good idea to use `assert` statement for testing your functions, example:
```
assert fibonacci(9) == 34
```

**Pull request merged**